HolyshiftService/Auth/JwtService.cs

```diff
@@ -0,0 +1,76 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using HolyShift.Config;
+using HolyShift.Database.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+namespace HolyShift.Auth;
+
+public class JwtService
+{
+    private readonly IOptions<AuthConfig> _authConfig;
+    private readonly HolyShiftDbContext _db;
+
+    public JwtService(IOptions<AuthConfig> authConfig, HolyShiftDbContext db)
```

Comment:
You should never access the dbcontext outside the dao.
You should inject the dao and do any db access there.
Remember, separation of concerns and single responsibility principles.